src/test/java/io/cdap/plugin/hubspot/sink/etl/HubspotSinkTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2017 Cask Data, Inc.
+ * Copyright © 2020 Cask Data, Inc.
```

Comment:
nit: 2017-2022

Reply:
Mm, it is a file from 2020, so my guess is that this line was missed when submitting the original code. My intention was to fix that, but I'm not really familiar with how copyright updates work, should there be 2017-2022 then or?..

Reply:
Yup, it should be the range from the year which the file was created to the year which the file was last modified.

Reply:
Okay, reverted this change, since we don't seem to update the years in the modified files these days.